ghost/core/core/server/services/posts/stats/post-stats.js

```diff
@@ -0,0 +1,31 @@
+class PostStats {
+    /**
+     * @param {Object} config
+     * @param {Object} config.db - an instance holding knex connection to the database
+     * @param {Boolean} config.isSQLite - flag identifying if storage is connected to SQLite
+     */
+    constructor({db, isSQLite}) {
+        this._db = db;
+        this._isSQLite = isSQLite;
```

Comment:
No biggie, but I would consider not passing this in, and calculating it in this module.